bolt/test/AArch64/dummy-return.test

```diff
@@ -0,0 +1,13 @@
+// Tests that AArch64 is able to instrument static binaries in relocation mode,
+// by checking that '__bolt_fini_trampoline' is generated and contains a 'ret'
+// instruction.
+
+REQUIRES: system-linux
+
+RUN: %clang %p/../Inputs/main.c -o %t -Wl,-q -static
```

Comment:
```suggestion
RUN: %clang %cflags %p/../Inputs/main.c -o %t -Wl,-q -static
```

Reply:
Thanks. I applied the suggestion, however, this would break my current c test as it interferes with `-static`.
I will be converting the test to assembly with a follow-up patch.